PowerValidatedSolutions.psm1

```diff
@@ -32706,18 +32883,47 @@ Function Test-vRLIAuthentication {
 Export-ModuleMember -Function Test-vRLIAuthentication
 
 Function Test-vRAConnection {
+    <#
+        .SYNOPSIS
+        Check network connectivity to a vRealize Automation cluster or node.
+
+        .DESCRIPTION
+        Checks the network connectivity to a vRealize Automation cluster or node.
+        Supports testing a connection on ports 443 (HTTPS), 8080 (HTTP), 22 (SSH). Default: 443 (HTTPS).
+
+        .EXAMPLE
+        Test-vRAConnection -server xint-vrops01.rainpole.io
+        This example checks network connectivity with a vRealize Automation cluster or node on default port, 443 (HTTPS).
+
+        .EXAMPLE
+        Test-vRAConnection -server xint-vrops01.rainpole.io -port 443
```

Comment:
```suggestion
        Test-vRAConnection -server xint-vra01.rainpole.io -port 443
```

Reply:
Addressed in 14b2e73bdb02f296b8ecbcda1032260e254946ce.